server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -596,10 +596,12 @@ private void emitSwiftHeader(PodSpec pod, Map<String, Object> manifestContext, L
         frameworkPaths.addAll(getFrameworkPaths(pod.dir));
         frameworkPaths.addAll(resolvedPods.getFrameworksSearchPaths());
 
+        File sourceListFile = ExtenderUtil.writeSourceFilesListToTmpFile(pod.swiftSourceFilePaths);
+
         Map<String, Object> context = createContext(manifestContext);
         context.put("ext", ImmutableMap.of("includes", includes, "frameworks", frameworks, "frameworkPaths", frameworkPaths));
         context.put("moduleName", pod.moduleName);
-        context.put("swiftSourceFiles", pod.swiftSourceFilePaths);
+        context.put("swiftSourceFiles", String.format("@%s", sourceListFile.getAbsolutePath()));
```

Comment:
Here and below: in Mustache template list expected. But If pass value as String - it works. It feels a bit incorrect to me but it works.